lib/Dialect/Arc/Export/ExportModelInfo.cpp

```diff
@@ -0,0 +1,41 @@
+//===- ExportModelInfo.cpp - Exports model info to JSON format ------------===//
```

Comment:
Could we use the same name as the header file this file is implementing (or vice versa)?

Reply:
oops!